pkg/database/realm.go

```diff
@@ -109,6 +109,23 @@ var (
 
 	ENXRedirectDomain = os.Getenv("ENX_REDIRECT_DOMAIN")
 
+	AnomalyAllowedStdevs = func() float64 {
+		v := os.Getenv("ANOMALY_ALLOWED_STDEVS")
```

Comment:
if you add this to one of the config structs, I think startup will fail in a more graceful way (probably need to leave these here too)